extensions/wikia/Search/classes/Result/ResultHelper.php

```diff
@@ -33,8 +35,9 @@ public static function extendResult($result, $pos, $descWordLimit)
 			$imageURL = ImagesService::getImageSrcByTitle(
 				(new \CityVisualization)->getTargetWikiId($result['lang_s']),
 				$imageFileName,
-				WikiaSearchController::CROSS_WIKI_PROMO_THUMBNAIL_WIDTH,
-				WikiaSearchController::CROSS_WIKI_PROMO_THUMBNAIL_HEIGHT);
+				$imageSizes['width'],
+				$imageSizes['height']
+			);
 		}//TODO: end
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)